start-guide/quickstart/manageconnections.rst

```diff
@@ -208,6 +208,32 @@ This step-by-step example shows you how to connect to a device.
 
 .. tip:: Remember - you can put all of these commands into a single script. We'll show you how in the :ref:`parse-output` section. 
 
+
+Depends on your device terminal settings, when you connect to device using CLI and execute command, you would see "press any key to continue". 
+For humans, this point of brake, give possibility to analyse output. However from automation point of view it would break parsers, as they change output data. 
```

Comment:
A few grammar fixes

```
Depending on your device terminal settings, when you connect to a device using a CLI and execute a command, you would sometimes see "press any key to continue". 
For humans, this break-point gives a possibility to analyze output. However, from automation point of view it would break parsers, as they change output data. 
```